lib/open_referral_transformer.rb

```diff
@@ -94,6 +97,23 @@ def transform_locations
           collect_phone_data(phone_key: k, phone_hash: v, input: input)
         elsif v["model"] == "postal_address"
           collect_address_data(address_key: k, address_hash: v, input: input)
+        elsif v["model"] == "regular_schedule"
+          if input["Hours of operation"] != nil
```

Comment:
In Ruby, you don't need the `!= nil` -- you can just do: `if input["Hours of operation"]`. Presence (opposite of `nil`) evaluates to `true` 😸